decentraland/unity-explorer
# feat: implemented extended-urn wearable indexing & publish of profile from backpack

Implements wearable indexing by extended-urns & shortened-urns. Wearables for the current user will be requested with extended-urns. Allows the saving of the profile from the backpack, using extended-urns within the profile.

# How to test?

1. Open the backpack
2. Equip new wearables
3. Close the backpack
4. Restart the client
5. Check that all saved changes are persisted
